topsort-endpoints.yml

```diff
@@ -199,7 +237,7 @@ components:
       type: object
       required:
         - eventType
-        - session
```

Comment:
You cannot remove the `session` as it is required for non promoted items.

Also in the case of promoted items we may still want to have the session. One reason is consistency and the other is to prevent cases when the auction was created for one user but shown to another.